heron/cli/src/python/.heronrc.sample

```diff
@@ -0,0 +1,10 @@
+*:*:* --config-property test-hello-global   #global
```

Comment:
changed to heronrc.sample
the argparser_unittest.py also changed to heronrc.test.